llpc/builder/llpcBuilderImplImage.cpp

```diff
@@ -929,19 +939,24 @@ Value* BuilderImplImage::CreateImageLoad(
                             derivatives);
 
     // The intrinsics insist on an FP data type, so we need to bitcast if we want an integer data type.
-    Type* pIntrinsicDataTy = pResultTy;
     if (pTexelTy->isIntOrIntVectorTy())
     {
-        pIntrinsicDataTy = ConvertToFloatingPointType(pTexelTy);
-        if (pTexelTy != pResultTy)
-        {
-            // TFE
-            pIntrinsicDataTy = StructType::get(pTexelTy->getContext(), { pTexelTy, getInt32Ty() });
-        }
+        pTexelTy = ConvertToFloatingPointType(pTexelTy);
+    }
+
+    Type* pIntrinsicDataTy = nullptr;
+    if (isa<StructType>(pResultTy))
+    {
+        // TFE
```

Comment:
What does TFE stand for?

Reply:
TFE stands for "texel fail enable" for partial texture residency. This is a hardware term for sparse texturing. Here, we align the naming with LLVM backend and use this term. Admittedly, it is somewhat confusing if code readers are not very familiar with AMD hardware:) We can later make a correction commit to modify the comments and use "sparse texturing" instaed.